mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -1,67 +1,69 @@
 CLASS net/minecraft/class_3620 net/minecraft/block/MapColor
 	COMMENT Represents the surface color of a block when rendered from the {@link net.minecraft.client.gui.MapRenderer}.
-	FIELD field_15976 CLAY Lnet/minecraft/class_3620;
+	COMMENT Color names refer to a material or an object which refers to their vanilla Minecraft textures, not their real-world counterparts, eg. "emerald green".
+	COMMENT Names are in the form of <i>blockReference_baseColor</i>.
```

Comment:
Slightly incorrect as this doesn't apply to all of them. (for example, `BROWN` or `OFF_WHITE`)

Reply:
Uh, why was this resolved? It's still not true as all names are not in that form.

Reply:
Should be clearer now